gluster-exporter/metric_volume_counts.go

```diff
@@ -124,6 +132,29 @@ func volumeCounts() error {
 	return nil
 }
 
+func volumeStatus() error {
+	isLeader, err := gluster.IsLeader()
+	if !isLeader || err != nil {
+		// Unable to find out if the current node is leader
+		// Cannot register volume metrics at this node
+		log.WithError(err).Error("Unable to find if the current node is leader")
+		return err
+	}
+	volumes, err := gluster.VolumeInfo()
+	if err != nil {
+		return err
+	}
+	for _, volume := range volumes {
+		up := 0
+		if volume.State == "Started" {
```

Comment:
we are already providing count of started volumes. https://github.com/gluster/gluster-prometheus/blob/master/docs/metrics.adoc#gluster_volume_started_count

Please include this metrics there itself. Otherwise we will end up calling volume info multiple times

Alternatively can we only export this metrics and derive the started volume count?

Reply:
Will collect this metric while collecting started count. Its better as we don't need to expose another collector itself.